docs/source/en/model_doc/layoutlmv2.mdx

```diff
@@ -44,13 +44,15 @@ including FUNSD (0.7895 -> 0.8420), CORD (0.9493 -> 0.9601), SROIE (0.9524 -> 0.
 RVL-CDIP (0.9443 -> 0.9564), and DocVQA (0.7295 -> 0.8672). The pre-trained LayoutLMv2 model is publicly available at
 this https URL.*
 
-LayoutLMv2 depends on `detectron2`, `torchvision` and `tesseract`. Run the
-following to install them: 
+LayoutLMv2 depends on `detectron2`. Run the following to install it: 
 ```
 python -m pip install 'git+https://github.com/facebookresearch/detectron2.git'
+```
+If you wish to use the OCR feature, you should also install `torchvision` and `tesseract`:
+```
 python -m pip install torchvision tesseract
 ```
-(If you are developing for LayoutLMv2, note that passing the doctests also requires the installation of these packages.)
+(If you are developing for LayoutLMv2, note that passing the doctests and certain tests for LayoutLMv2 requires the installation of all three packages.)
```

Comment:
Hi @NielsRogge , your second point is valid. I changed it to `pip install pytesseract` (although I'm not sure if we should include how to install tesseract itself on Ubuntu as it seems a bit too specific?)

Regarding your first point, you can try removing `torchvision` and running the doctests. Run from the root directory: 

```
python utils/prepare_for_doc_test.py src/transformers/utils/doc.py src/transformers/models/layoutlmv2/modeling_layoutlmv2.py

pytest --doctest-modules src/transformers/models/layoutlmv2/modeling_layoutlmv2.py -sv --doctest-continue-on-failure
```
The doctests will fail and prompt you to install `torchvision`. This is because`torchvision` is a dependency of detectron2, but running python -m pip install 'git+https://github.com/facebookresearch/detectron2.git’ won’t automatically install `torchvision`. 

In addition, we should also install both `torch` and `datasets` for the doctests to pass. 

If any of the above packages is missing, doctests will fail.

Reply:
@NielsRogge Perhaps this is not the most high priority issue but it should be an easy fix. I'm also ok with removing this entire paragraph altogether and let the terminal errors prompt the users to install all the packages as necessary.